IFC4x3/Properties/e/Emodulus_10zwR$ze1ElBZzkStdNfro/DocProperty.xml

```diff
@@ -0,0 +1,3 @@
+<?xml version="1.0" encoding="utf-8"?>
+<DocProperty xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" id="Emodulus_10zwRze1ElBZzkStdNfro" Name="Emodulus" UniqueId="40f7a6ff-f680-4ebc-b8fd-b9cde75e9d72" PrimaryDataType="IfcModulusOfElasticityMeasure" />
```

Comment:
Reuse existing "YoungModulus".

Reply:
Fixed in [e263b67](https://github.com/bSI-InfraRoom/IFC-Specification/pull/630/commits/e263b67252fc5988e33bf8c5c130de5ca8fc56a8)